AdobeStockClient/etc/config.xml

```diff
@@ -15,7 +15,7 @@
                 <environment>PROD</environment>
                 <product_name>magento-adobe-stock-integration</product_name>
                 <token_url>https://ims-na1.adobelogin.com/ims/token</token_url>
-                <auth_url_pattern><![CDATA[https://ims-na1.adobelogin.com/ims/authorize?client_id=#{client_id}&scope=openid&response_type=code]]></auth_url_pattern>
+                <auth_url_pattern><![CDATA[https://ims-na1.adobelogin.com/ims/authorize?client_id=#{client_id}&redirect_uri=#{redirect_uri}&locale=en_US&scope=openid&response_type=code]]></auth_url_pattern>
```

Comment:
I have added the locale here. Is it correct?

Reply:
@konarshankar07 Based on description here https://github.com/magento/adobe-stock-integration/issues/41 locale should takes from Magento configuration

Reply:
@Den4ik 
Thanks for the clarification. I'll add the Magento configuration locale.